atintegrators/SimpleRadiationPass.c

```diff
@@ -0,0 +1,189 @@
+
+#include "atelem.c"
+#include "atrandom.c"
+
+struct elem 
+{
+  double *damp_mat;
+  double betax;
+  double betay;
+  double alphax;
+  double alphay;
+  double dispx;
+  double dispxp;
+  double dispy;
+  double dispyp;
+  double U0;
+  double EnergyLossFactor;
+};
+
+void SimpleRadiationPass(double *r_in,
+           double *damp_mat, double betax, double alphax,
+           double betay, double alphay, double dispx, double dispxp,
+           double dispy, double dispyp, double EnergyLossFactor, int num_particles)
+
+{
+  double *r6;
+  int c, i;
+  double *dmatx = damp_mat;
+  double *dmatxp = damp_mat + 6;
+  double *dmaty = damp_mat + 2*6;
+  double *dmatyp = damp_mat + 3*6;
+  double *dmatdp = damp_mat + 4*6;
+  double *dmatz = damp_mat + 5*6;
+  double x,xp,y,yp,z,dpp;
+  double X,XP,Y,YP;
+  
+  #pragma omp parallel for if (num_particles > OMP_PARTICLE_THRESHOLD*10) default(shared) shared(r_in,num_particles) private(c,r6)
+  for (c = 0; c<num_particles; c++) { /*Loop over particles  */
+    r6 = r_in+c*6;
+    
+    if(!atIsNaN(r6[0])) {
+
+      dpp = r6[4];
+      x = r6[0] - dispx*dpp;
+      xp = r6[1] - dispxp*dpp;
+      
+      y = r6[2] - dispy*dpp;
+      yp = r6[3] - dispyp*dpp;
+      
+      X = x/sqrt(betax);
+      XP = xp*sqrt(betax) + x*alphax/sqrt(betax);
+      Y = y/sqrt(betay);
+      YP = yp*sqrt(betay) + y*alphay/sqrt(betay);
+
+      z = r6[5];        
+    
+      if(dmatx[0]!=1.0) {
+        X *= dmatx[0];
+      }
+
+      if(dmatxp[1]!=1.0) {
+        XP *= dmatxp[1];
+      }
+      
+      if(dmaty[2]!=1.0) {
+        Y *= dmaty[2];
+      }
+
+      if(dmatyp[3]!=1.0) {
+        YP *= dmatyp[3];
+      }
+      
+      if(dmatdp[4]!=1.0) {
+        dpp *= dmatdp[4];
+      }
+
+      if(dmatz[5]!=1.0) {
+        z *= dmatz[5];
+      }
+
+      r6[4] = dpp;
+
+      x = X*sqrt(betax);

```

Comment:
I removed beta and alpha completely. Keeping dispersion as it will be needed when we eventually redo fast ring using this passmethod. 

All other changes implemented.